### PR TITLE
adjust search filters

### LINKED ISSUE #467: :iphone: Adjust user search page with filters
### How it should look:
Results

### How it looks like
image

### Figma design
https://www.figma.com/file/pxj9Hhk5AdOxXWJP0dq4gT/Library-App-Spec?node-id=4993%3A15165

### PR DESCRIPTION
**Pull request by mabrur-h**
Close #467 
image
image

### CODE REVIEW

**Review by Chichmarin17 — CHANGES_REQUESTED** [review]
Can you adjust `back to all categories` button to match design as well?

**Comment by mabrur-h** [comment]
> Can you adjust `back to all categories` button to match design as well?

Close #467
image
image

I added margin to the back button

**Review by Chichmarin17 — APPROVED** [review]

File: packages/frontapp/src/app/components/BooksByCategory/tags.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { FC } from 'react';
 import { useSearchParams } from 'react-router-dom';
 import { ReactComponent as Close } from '../../../assets/Close.svg';
 import { useTranslation } from 'react-i18next';
+import { dimensions } from '@mimir/ui-kit';
 
 const Header = styled.h2`
   font-size: 24px;
```
Comment (sabrina34352) [inline packages/frontapp/src/app/components/BooksByCategory/tags.tsx:9]: use `dimensions` toolkit variables

File: packages/frontapp/src/app/components/BooksByCategory/tags.tsx [hunk 2]
```diff
@@ -28,6 +29,15 @@ const TagsWrapper = styled.div`
   display: flex;
   flex-wrap: wrap;
   gap: 16px;
```
Comment (sabrina34352) [inline packages/frontapp/src/app/components/BooksByCategory/tags.tsx:31]: same thing

**Review by sabrina34352 — CHANGES_REQUESTED** [review]

File: packages/frontapp/src/app/components/BackButton/index.tsx [hunk 3]
```diff
@@ -15,6 +15,7 @@ export const ButtonWrapper = styled.div`
 export const GoBack = styled.a`
   font-weight: 600;
   font-size: ${dimensions.base};
+  margin-left: 18px;
```
Comment (sabrina34352) [inline packages/frontapp/src/app/components/BackButton/index.tsx:18]: do not forget to change them to dimesions.ts

**Review by sabrina34352 — APPROVED** [review]

**Comment by ngame2011** [comment]
@mabrur-h @Chichmarin17 what is the status of this PR now? I see that it is necessary to make minor changes and you can close this PR. 10 days for such changes is too long.

**Comment by mabrur-h** [comment]
@ngame2011 I think we can close this PR. I have changed px to constant values.

**Pull request merged**